test/utils.sh

```diff
@@ -222,24 +222,26 @@ cli_get_pods_test_env () {
 
 test_secret_is_provided () {
   secret_value=$1
+  variable_name="${2:-secrets/test_secret}"
```

Comment:
What is `#:-`?

`variable_name=secrets/test_secret` is the second parm wasn't supplied?

Reply:
yes. it checks if `$2` exists and if not sets to a default value